app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -1140,7 +1140,7 @@ class SectionRenderingService(
           case Some(upscanData) =>
             val fileUploadName = "file"
             val otherAttributes = Map("form" -> upscanData.formMetaData.htmlId)
-            val attributes = Map("upscan" -> "upscan") ++ otherAttributes
+            val attributes = Map("data-upscan" -> "upscan") ++ otherAttributes
```

Comment:
Is this attribute actually used? I don't see any js changes, so why we would need it?

Reply:
This attribute is not actually used in js. I will remove it.